### PR TITLE
Show label summary more fancy

### PR DESCRIPTION
**Pull request by ferranb**

### CODE REVIEW

**Comment by ferranb** [comment]
He canviat el resum per a que es vegi així. Per una banda, a `kingdom_name` falten (o no passa res) alguns valors. I, per una altra, estan totes les classes igual representades malgrat estiguin desbalancejades. Ho dic perquè no tinc clar si, ara mateix, cal  fer servir la [funció per estratificar](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html)


**kingdom_name**

    Missing label values found in train dataset: ['Other_euk', 'No_euk']
    Missing label values found in eval dataset: ['Other_euk', 'No_euk']
    Missing label values found in test dataset: ['Other_euk', 'No_euk']
                         train            eval             test      
    Metazoa          485091  99.86%   60644  99.87%   60640  99.86% 
    Viridiplantae       424   0.09%      41   0.07%      59   0.10% 
    Fungi               265   0.05%      37   0.06%      24   0.04% 
                     485780 100.00%   60722 100.00%   60723 100.00%

**phylum_name**

                           train            eval             test      
    Arthropoda         355641  73.21%   44446  73.20%   44455  73.21% 
    Chordata            77617  15.98%    9778  16.10%    9756  16.07% 
    Mollusca            29212   6.01%    3614   5.95%    3555   5.85% 
    Other_metazoa        7068   1.45%     912   1.50%     949   1.56% 
    Annelida             6053   1.25%     724   1.19%     741   1.22% 
    Echinodermata        5104   1.05%     632   1.04%     658   1.08% 
    Platyhelminthes      2261   0.47%     287   0.47%     266   0.44% 
    Cnidaria             2135   0.44%     251   0.41%     260   0.43% 
    No_metazoa            689   0.14%      78   0.13%      83   0.14% 
                       485780 100.00%   60722 100.00%   60723 100.00% 



**class_name**

                            train            eval             test      
    Insecta             300801  61.92%   37566  61.87%   37496  61.75% 
    No_arthropoda       130139  26.79%   16276  26.80%   16268  26.79% 
    Arachnida            23405   4.82%    2884   4.75%    2936   4.84% 
    Malacostraca         19519   4.02%    2424   3.99%    2499   4.12% 
    Hexanauplia           2904   0.60%     375   0.62%     359   0.59% 
    Thecostraca           2796   0.58%     387   0.64%     353   0.58% 
    Collembola            2793   0.57%     379   0.62%     335   0.55% 
    Branchiopoda          1622   0.33%     208   0.34%     228   0.38% 
    Chilopoda              565   0.12%      69   0.11%      90   0.15% 
    Diplopoda              486   0.10%      57   0.09%      68   0.11% 
    Ostracoda              455   0.09%      63   0.10%      63   0.10% 
    Pycnogonida            166   0.03%      18   0.03%      19   0.03% 
    Other_arthropoda       129   0.03%      16   0.03%       9   0.01% 
                        485780 100.00%   60722 100.00%   60723 100.00% 



**order_name**

                         train            eval             test      
    No_insecta       184979  38.08%   23156  38.13%   23227  38.25% 
    Lepidoptera      102787  21.16%   12850  21.16%   12711  20.93% 
    Diptera           61345  12.63%    7782  12.82%    7697  12.68% 
    Coleoptera        53757  11.07%    6602  10.87%    6533  10.76% 
    Hymenoptera       35306   7.27%    4508   7.42%    4421   7.28% 
    Hemiptera         17028   3.51%    2038   3.36%    2199   3.62% 
    Trichoptera        8134   1.67%    1037   1.71%    1024   1.69% 
    Orthoptera         5675   1.17%     640   1.05%     754   1.24% 
    Odonata            3728   0.77%     452   0.74%     507   0.83% 
    Ephemeroptera      3649   0.75%     434   0.71%     480   0.79% 
    Other_insecta      2564   0.53%     338   0.56%     334   0.55% 
    Plecoptera         2518   0.52%     333   0.55%     312   0.51% 
    Neuroptera         1414   0.29%     195   0.32%     169   0.28% 
    Blattodea          1336   0.28%     160   0.26%     168   0.28% 
    Thysanoptera       1004   0.21%     120   0.20%     111   0.18% 
    Psocoptera          556   0.11%      77   0.13%      76   0.13% 
                     485780 100.00%   60722 100.00%   60723 100.00%

**Comment by ntorqulu** [comment]
Pel primer rank, és correcte. Després del preprocessing no queden Bacteria -> No_euk ni seqüències que no pertanyin a les classes [Metazoa, Viridiplantae, Fungi]
P.t. són normals els resultats.
Sobre l'estratificació, per ara no cal. Però quan baixem de rank i no imposem el filtratge de classe segurament l'haurem d'implementar.
